addons/website/tests/test_ui.py

```diff
@@ -104,7 +104,7 @@ def test_html_editor_scss(self):
                 self.env.ref('website.group_website_designer').id
             ])]
         })
-        self.start_tour("/", 'test_html_editor_scss', login='admin')
+        self.start_tour("/", 'test_html_editor_scss', login='admin', timeout=120)
```

Comment:
Do we actually need this since you added it on the specific steps needing it?

Reply:
The full scenario contains 5 SCSS updates - when the server is heavily loaded each of them might approach 10 seconds, and we also need the time to run the other steps. This is getting very close to the default 60 seconds which is the reason I doubled it.

Reply:
Thanks, that seems fine then :+1: